root/static/scripts/edit/URLCleanup.js

```diff
@@ -1876,14 +1876,16 @@ const CLEANUPS = {
     match: [new RegExp('^(https?://)?www\\.muziekweb\\.(eu|nl)/', 'i')],
     type: LINK_TYPES.otherdatabases,
     clean: function (url) {
-      return url.replace(/^(?:https?:\/\/)?(?:www\.)?muziekweb\.(?:eu|nl)\/(?:[a-z]{2}\/)?Link\/([A-Z]{1,3}\d+).*$/, 'https://www.muziekweb.eu/Link/$1/');
+      return url.replace(/^(?:https?:\/\/)?(?:www\.)?muziekweb\.(?:eu|nl)\/(?:[a-z]{2}\/)?Link\/([A-Z]{1,3}\d+)(?:\/)?((CLASSICAL(?:\/COMPOSER)?|POPULAR)\/)?.*$/, 'https://www.muziekweb.nl/Link/$1/$2');
```

Comment:
Old domain `.com` should probably be matched (above) and cleaned up (here) too.

The RegExp can be simplified to use less groups, and it seems slightly clearer about handling slashes:
```suggestion
      return url.replace(/^(?:https?:\/\/)?(?:www\.)?muziekweb\.(?:com|eu|nl)\/(?:[a-z]{2}\/)?Link\/([A-Z]{1,3}\d+(?:\/(?:CLASSICAL(?:\/COMPOSER)?|POPULAR))?).*$/, 'https://www.muziekweb.nl/Link/$1/');
```